sentry-android-core/src/main/java/io/sentry/android/core/ContextUtils.java

```diff
@@ -45,4 +46,24 @@ static String getVersionCode(final @NotNull PackageInfo packageInfo) {
   private static @NotNull String getVersionCodeDep(final @NotNull PackageInfo packageInfo) {
     return Integer.toString(packageInfo.versionCode);
   }
+
+  /**
+   * Returns the Settings.Secure.ANDROID_ID and if not valid, fallback to defaultValue
+   *
+   * @param defaultValue the defaultValue
+   * @return Settings.Secure.ANDROID_ID if valid or defaultValue
+   */
+  @SuppressWarnings("HardwareIds")
+  static @Nullable String getAndroidId(
+      final @NotNull Context context, final @Nullable String defaultValue) {
+    String androidId =
+        Settings.Secure.getString(context.getContentResolver(), Settings.Secure.ANDROID_ID);
+
+    if (androidId == null
+        || androidId.isEmpty()
+        || androidId.equalsIgnoreCase("9774d56d682e549c")) {
```

Comment:
Could we maybe add a comment about what `9774d56d682e549c` is? I had to Google it ;)

Reply:
ups, copy-pasta mistake.
the original code has a comment which leads to the google blogpost
see https://github.com/getsentry/sentry-android/blob/19aaac5ada0d3671501b0959db59437173db96ee/sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java#L869